src/selectors/per-thread/markers.js

```diff
@@ -414,6 +414,7 @@ export function getMarkerSelectorsPerThread(
       ProfileSelectors.getMarkerSchema,
       ProfileSelectors.getMarkerSchemaByName,
       ProfileSelectors.getCategories,
+      threadSelectors.getThread,
```

Comment:
I don't think we use anything other than the `stringTable` field of the thread. Can we directly pass only `getStringTable` instead of `getThread` here and below?

Reply:
Good point! We can also then change the definition of ParseLabel to just take a stringtable, which should make it clearer why we need that extra argument.